caffe2/requirements.txt

```diff
@@ -3,3 +3,4 @@ enum34
 pyyaml
 requests
 typing
+torchvision
```

Comment:
It doesn't look appropriate to me. I would install the torchvision in the test.sh instead.

Reply:
Fixed